### PR TITLE
[webui] Get rid of ActiveXML in PatchinfoController#save

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

File: src/api/app/controllers/webui/patchinfo_controller.rb [hunk 1]
```diff
@@ -153,8 +152,14 @@ def save
...
         begin
-          frontend.put_file(xml, project: @project,
-                                 package: @package, filename: filename)
+          authorize @package, :update?
+
+          Package.verify_file!(@package, '_patchinfo', xml)
+
+          Suse::Backend.put @package.source_path('_patchinfo', user: User.current.login), xml
+
+          @package.sources_changed(wait_for_update: true) # wait for indexing for special files
```
Comment (mdeniz) [inline src/api/app/controllers/webui/patchinfo_controller.rb]: @adrianschroeter that line is the only thing that I'm not sure of... can you review it please?

**Comment by hennevogel** [comment]
Please also delete this method from the `do_backend_login` before filter

**Comment by hennevogel** [comment]
Otherwise, it looks good to me!

**Comment by mdeniz** [comment]
Done! :laughing: :+1:

**Comment by hennevogel** [comment]
Sorry, now you need to rebase ;-)

**Comment by mdeniz** [comment]
Done again :smiley:

**Comment by hennevogel** [comment]
Alrighty! :+1:

**Pull request merged**